clusters/temp-privatek8s.yaml

```diff
@@ -4,22 +4,17 @@ helmDefaults:
   timeout: 300
   wait: true
 repositories:
+  - name: datadog
+    url: https://helm.datadoghq.com
+  - name: ingress-nginx
+    url: https://kubernetes.github.io/ingress-nginx
   - name: jenkins
     url: https://charts.jenkins.io
   - name: jenkins-infra
     url: https://jenkins-infra.github.io/helm-charts
-  - name: ingress-nginx
-    url: https://kubernetes.github.io/ingress-nginx
   - name: jetstack
     url: https://charts.jetstack.io
 releases:
-  - name: cert-manager
```

Comment:
New:
```
  - name: datadog
    namespace: datadog
    chart: datadog/datadog
    version: 2.36.4
    timeout: 840 # Should be <= 14 min (because the job runs every 15 min)
    values:
      - "../config/ext_datadog.yaml.gotmpl"
      - "../config/ext_temp-privatek8s-datadog.yaml"
    secrets:
      - "../secrets/config/datadog/temp-privatek8s-secrets.yaml"
```